functions/Get-DbaDatabaseFreespace.ps1

```diff
@@ -108,6 +108,12 @@ Returns database files and free space information for the db1 and db2 on localho
 			$server = Connect-SqlServer $s -SqlCredential $SqlCredential
 			#If IncludeSystemDBs is true, include systemdbs
 			#only look at online databases (Status equal normal)
+
+			if ($server.VersionMajor -eq '8'){
```

Comment:
While this works, it should be 8 instead of '8' because that is a string of 8. Will fix after merge.